tests/analysis/test_hypothesis.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import pytest
+
+from cluster_experiments.experiment_analysis import (
+    ClusteredOLSAnalysis,
+    GeeExperimentAnalysis,
+    MLMExperimentAnalysis,
+    OLSAnalysis,
+    TTestClusteredAnalysis,
+)
+from tests.examples import analysis_df, generate_clustered_data
+
+
+@pytest.mark.parametrize("hypothesis", ["less", "greater", "two-sided"])
+@pytest.mark.parametrize("analysis_class", [OLSAnalysis])
+def test_get_pvalue_hypothesis(analysis_class, hypothesis):
```

Comment:
I would also add some test where you initialize power from_dict and check that the hypothesis change is doing something